addons/payment/controllers/post_processing.py

```diff
@@ -63,7 +64,6 @@ def poll_status(self, **_kwargs):
                 )
                 raise
 
-        # Return the post-processing values to display the transaction summary to the customer.
```

Comment:
If the comment is no longer true, it should mean that we can get rid of some post-processing values.

Reply:
The caller of `/payment/status/poll` indeed only uses `provider_code`, `state` and `landing_route`. However, the caller of `/payment/status` uses all the values returned by `_get_post_processing_values` (except `provider_code` and `provider_name`).

I don't think it's worth having separate versions of `_get_post_processing_values` for the 2 RPCs, but I think it would be a good idea to make `/payment/status/poll` only return the 3 values that are actually used. WDYT?

And I'd remove `provider_name` from `_get_post_processing_values` since it's unused.

Reply:
Following the suggestion above to unify the templates, you'd need to base your `t-out`s on the transaction record. Hence, I assume it would make /payment/status less dependent on `_get_post_processing_values`.

Reply:
I removed `_get_post_processing_values` altogether as `/payment/status` now uses the transaction directly (instead of the post processing values) and `/payment/status/poll` only needs 3 values so we can simply create a dict on the spot.

FYI, `_get_post_processing_values` was only used by `/payment/status` and `/payment/status/poll`.